client/components/inline-support-link/README.md

```diff
@@ -0,0 +1,30 @@
+Inline Support Link
+===================
+
+This component displays a link and icon. When clicked, it opens a SupportArticleDialog with an ExternalLink to the article if supplied.
+
+## Example Usage:
+
+```js
+import InlineSupportLink from 'components/inline-support-link';
+
+render() {
+	const inlineSupportProps = {
+		text: 'Learn more about Podcasting',
+		supportLink: 'https://en.support.wordpress.com/audio/podcasting/',
+		supportPostId: 'https://example.com/#privacy',
```

Comment:
Is this a string intentionally? I wonder if it could be misleading if so since we expect a number (`38147` in the devdocs example)

Reply:
Nice catch - this was just a c/p fail from another README I grabbed.